lib/sonice/public/script.js

```diff
@@ -116,6 +116,19 @@ $(function() {
     })
   }
 
+  // Toggle the playpause button on the remote button widget
+  var togglePlayPause = function(toggle_playpause_button){
```

Comment:
Oh I see.  Thanks @sunny.

That clears things up.

I'll change it anyways.  So that I build up good habits of writing good coding practices for open source projects.

Cheers!